pglast/enums/__init__.py

```diff
@@ -7,6 +7,7 @@
 #
 
 # include/catalog
+from .pg_am import *     # noqa
```

Comment:
Remember to add the new `pg_am.rst` document in `docs/enums.rst`

Reply:
Done ! Thank you.